AI/Nullkiller/Pathfinding/AINodeStorage.h

```diff
@@ -27,8 +27,8 @@ namespace NKAI
 {
 namespace AIPathfinding
 {
-	const int BUCKET_COUNT = 3;
-	const int BUCKET_SIZE = 7;
+	const int BUCKET_COUNT = 1;
```

Comment:
This should be fetched from settings if compiler can allow it or should be left as is for now. Or you can change it according to AI_TRACE_LEVEL >= 1 for debug purposes like I did for parallelism.

Reply:
@nullkiller should we merge with this change?

Reply:
No I do not think we should include it in this form. This configuration turn hashmap into plain vector. This reduce certain random behavior as it no longer depends on Actor address but may hurt performance a bit. When I introduced it the impact of searching through plain vector seamed to be significant according to profiler.
The best option is to make it configurable, Reducing number will reduce memory footprint. Some 1-10 configuraiton for instance